hazelcast/src/test/java/com/hazelcast/client/config/YamlClientConfigBuilderTest.java

```diff
@@ -665,6 +668,21 @@ public void testPersistentMemoryConfiguration_NotExistingModeThrows() {
         buildConfig(yaml);
     }
 
+    @Test
+    @Override
+    public void testNativeMemoryConfiguration_isBackwardCompatible() {
```

Comment:
similar comment to yaml test file above

Reply:
1135d7cd77ef1592681bdc17a0779a1f4d21915e